src/commands/watch.rs

```diff
@@ -93,6 +94,7 @@ impl WatchCommand {
 
         let spin_bin = std::env::current_exe()?;
         let manifest_file = spin_common::paths::resolve_manifest_file_path(&self.app_source)?;
+        let manifest_file = manifest_file.absolutize()?.to_path_buf(); // or watchexec misses files in subdirectories
```

Comment:
BUT WHY

Reply:
BECAUSE REASONS LANN REASONS

We could probably absolutise at the last moment before we pass the directory to `watchexec` - I just get paranoid about capturing the true location as early as possible (a) so I don't need to remember to do it at _all_ the last moments and (b) in case some other piece of code sneakily switches out the current directory from underneath me.  _thousand yard stare_

We can iterate on this for sure though.